source/guide/3.2/reference-builtin-workflows.md

```diff
@@ -192,3 +192,49 @@ This sub-graph determines the operations that will be executed during the workfl
 * The following node instances will be re-installed: `war_1`, `webserver_1` and `webserver_host_1`.
 * The following relationships will be re-established: `war_1` **connected to** `database_1` and `webserver_host_1` **connected to** `floating_ip_1`.
 {%endnote%}
+
+# Scale
+
```

Comment:
i think scaling and healing should have a separate page for more detailed explanations. as far as the built-in workflows page this looks ok to me.